spec.yaml

```diff
@@ -759,3 +762,75 @@ paths:
           description: Error
           schema:
             $ref: "definitions.yaml#/definitions/V4GenericResponse"
+
+  /v4/releases/:
+    get:
+      operationId: getReleases
+      tags:
+        - releases
+      summary: Get releases
+      description: |
+        Lists all releases available for new clusters or for upgrading existing
+        clusters. Might also serve as an archive to obtain details on older
+        releases.
+      parameters:
+        - $ref: 'parameters.yaml#/parameters/OrganizationIdPathParameter'
+      responses:
+        "200":
+          description: Releases list
+          schema:
+            type: array
+            items:
+              $ref: "definitions.yaml#/definitions/V4ReleaseListItem"
+          examples:
+            application/json:
+              [
+                {
+                  "version": "1.14.9",
+                  "timestamp": "2017-09-21T08:14:03.37759Z",
+                  "description": "Minor updates of kubernetes and calico, both for security fixes.",
+                  "changelog": [
+                    {
+                      "component": "kubernetes",
+                      "description": "Security fixes"
+                    },
+                    {
+                      "component": "calico",
+                      "description": "Security fixes"
+                    }
+                  ],
+                  "components": [
+                    {
+                      "name": "kubernetes",
+                      "version": "1.5.8"
+                    },
+                    {
+                      "name": "calico",
+                      "version": "0.9.1"
+                    }
+                  ],
+                  "deprecated": true
+                },
+                {
+                  "version": "2.8.4",
+                  "timestamp": "2017-11-11T12:24:56.59969Z",
+                  "description": "Minor update of calico, fixing a bug.",
+                  "changelog": [
```

Comment:
I noticed this should be plural to be aligned with components.

Reply:
I wouldn't do that. According to my understanding, a changelog is one changelog, even if it has multiple items in it. Otherwise it should be named "changes" or something like that.

Reply:
Components is a list of things. It is descried in plural form. Changelogs is a ist of things. It is described in singular form. That does not make any sense to me. Do what you think is right. Disagree and commit.